lib/MusicBrainz/Server/Report/ArtistsThatMayBePersons.pm

```diff
@@ -12,31 +12,31 @@ WITH groups_entity0 AS (
          JOIN l_artist_artist laa ON laa.entity0 = artist.id
          JOIN link ON link.id = laa.link
          JOIN link_type ON link_type.id = link.link_type
-         WHERE artist.type IS DISTINCT FROM 1
+         WHERE artist.type NOT IN (1, 4)
```

Comment:
`null is distinct from 1` will return true, but `null not in (1, 4)` will return null, so this would appear to now exclude all artists with null types?

Reply:
Hmm, I did not know that, will add OR IS NULL :)